src/main/java/io/getunleash/repository/YggdrasilAdapters.java

```diff
@@ -69,8 +69,8 @@ public static Context adapt(UnleashContext context) {
         mapped.setRemoteAddress(context.getRemoteAddress().orElse(null));
         mapped.setProperties(context.getProperties());
         mapped.setCurrentTime(
-                DateTimeFormatter.ISO_DATE_TIME.format(
-                        context.getCurrentTime().orElse(ZonedDateTime.now())));
+                DateTimeFormatter.ISO_INSTANT.format(
```

Comment:
The Java date formatter will insert full timezones decscriptors (e.g. +02:00[Africa/Johannesburg]), which is Java's idea of a valid ISO 8601 timestamp, which Yggdrasil does not like. So we convert it to a Zulu time before sending it over